pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -105,6 +105,25 @@ func (p Provisioner) Provision(options *apibkt.BucketOptions) (*bktv1alpha1.Obje
 	}
 	logger.Infof("set user %q bucket max to %d", p.cephUserName, maxBuckets)
 
+	maxObjects := options.ObjectBucketClaim.Spec.AdditionalConfig["maxObjects"]
+	maxSize := options.ObjectBucketClaim.Spec.AdditionalConfig["maxSize"]
+	if maxObjects != "" || maxSize != "" {
+		if maxObjects != "" {
+			_, errCode, err = cephObject.SetQuotaUserObjectMax(p.objectContext, p.cephUserName, maxObjects)
```

Comment:
If it fails, the error must be logged so we can see what failed.

Reply:
will address in next PR

Reply:
Done